engine/cdn/objectstore/types.go

```diff
@@ -0,0 +1,7 @@
+package objectstore
+
+//Object is the interface for stuff needed to be stored in object store
+type Object interface {
```

Comment:
is this still useful ?

Reply:
Yes I use it for each type of object to store. It's very handy